File: Source/CslaExtremeDemos.BusinessVB/Roles.vb
Imports System.ComponentModel

Namespace CslaExtremeDemos.BusinessVB
    Public Enum Roles As Byte

        <Description("<Select role>")>
        Empty = 0
        <Description("Senior Developer")>
        Clerk
        <Description("Team Leader")>
        Chief
        <Description("Project Manager")>
        Governor
        <Description("Chief Executive Officer")>
        Boss
    End Enum
End Namespace

File: CslaExtremeDemos/CslaExtremeDemos.Business/Roles.cs
using System.ComponentModel;

namespace CslaExtremeDemos.Business
{
    public enum Roles : byte
    {
        [Description("<Select role>")]
        Empty = 0,
        [Description("Senior Developer")]
        Clerk,
        [Description("Team Leader")]
        Chief,
        [Description("Project Manager")]
        Governor,
        [Description("Chief Executive Officer")]
        Boss
    }
}